scripts/restore

```diff
@@ -59,74 +57,77 @@ ynh_restore_file "/etc/nginx/conf.d/$domain.d/$app.conf"
 #=================================================
 # RESTORE THE APP MAIN DIR
 #=================================================
-
 ynh_script_progression --message="Restoring files"
+
 ynh_restore_file "$final_path"
 
 #=================================================
 # RECREATE THE DEDICATED USER
 #=================================================
-
-# Create the dedicated user (if not existing)
 ynh_script_progression --message="Creating system user"
+
 ynh_system_user_create "$app"
 
 #=================================================
 # RESTORE USER RIGHTS
 #=================================================
 
 # Restore permissions on app files
-chown -R "$app": "$final_path"
+chown -R $app: "$final_path"
 chmod 600 "$final_path/config.ini"
 
 #=================================================
 # SPECIFIC RESTORATION
 #=================================================
 # REINSTALL DEPENDENCIES
 #=================================================
-
 ynh_script_progression --message="Installing app dependencies"
-ynh_install_app_dependencies $pkg_dependencies
+
+ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
 
 #=================================================
 # INSTALL NODEJS
 #=================================================
-
 ynh_script_progression --message="Installing nodejs"
-ynh_install_nodejs 12
+
+ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
 
 #=================================================
 # RESTORE THE POSTGRESQL DATABASE
 #=================================================
-
 ynh_script_progression --message="Restoring database"
+
 ynh_psql_test_if_first_run
 ynh_print_OFF
-ynh_psql_setup_db "$db_user" "$db_name" "$db_pwd"
+ynh_psql_setup_db --db_user=$db_user --db_name=$db_name --db_pwd=$db_pwd
```

Comment:
Are we sure there won't be any whitespace in the variables?

Reply:
I am sticking to the **example_ynh** template. We normally don't get spaces in thos variables...
https://github.com/YunoHost/example_ynh/blob/25251b00d316c41ed1fef6df90312df60615b8da/scripts/restore#L107-L110

Also I don't know if `ynh_print_OFF` `ynh_print_ON` are useful here ...